src/qt/bitcoingui.h

```diff
@@ -232,8 +234,8 @@ private Q_SLOTS:
     void detectShutdown();
 
     /** Show progress dialog e.g. for verifychain */
-    void showProgress(const QString &title, int nProgress);
-    
+    void showProgress(const QString &title, int nProgress, bool resume_possible, const std::function<void()>& cancel);
```

Comment:
Would be nice to have nullptr as a default for cancel.

Reply:
This was suggested before and I was not able to get it to work.